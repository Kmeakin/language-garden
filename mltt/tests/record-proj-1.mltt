let R : Type := {
    A : Type;
    a : A;
};

let r : R := {
    A := Type;
    a := {};
};

r.a
