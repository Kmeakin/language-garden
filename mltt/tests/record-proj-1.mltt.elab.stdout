<input> : Type :=
  let _ : Type := { A : Type; a : @0; }; let _ : @0 := {
                                             A := Type;
                                             a := {};
                                           };
                                           @0.a
